app/models/annotations/task.rb

```diff
@@ -50,7 +50,7 @@ def status
   end
 
   def project
-    self&.annotated&.project
+    self&.annotated&.projects&.first
```

Comment:
I would expect to remove this field completely

Reply:
Done